README.md

```diff
@@ -1,54 +1,201 @@
+[![Windows prereqs | GitHub](https://img.shields.io/badge/Windows_prereqs-GitHub-darkred.svg?logo=github&logoColor=333333&style=popout)](https://github.com/TASVideos/BizHawk-Prereqs/releases/latest)
 
-[Release Notes](http://tasvideos.org/Bizhawk/ReleaseHistory.html)
+A multi-system emulator written in C#. As well as quality-of-life features for casual players, it also has recording/playback and debugging tools, making it the first choice for TASers (Tool-Assisted Speedrunners).
 
-[Here](http://tasvideos.org/Bizhawk/Features.html) is a list of features offered by bizhawk.
+Click the "release" button above to grab the latest stable version ([changelog at TASVideos](http://tasvideos.org/Bizhawk/ReleaseHistory.html)). 
 
-## Download Binaries
+New users on Windows should click the "prereqs" button too, see *Installing* for info.
 
-Windows users, don't forget to run the [prereq installer](http://github.com/TASVideos/BizHawk-Prereqs/releases) first!
+**Never mix different versions** of BizHawk — Keep each version in its own folder.
 
-Release binaries can be found on [on github](http://github.com/TASVideos/BizHawk/releases)
+Jump to:
+* [Installing](https://github.com/TASVideos/BizHawk/blob/master/README.md#installing)
+	* [Windows 7/8.1/10](https://github.com/TASVideos/BizHawk/blob/master/README.md#windows-78110)
+	* [GNU+Linux and macOS](https://github.com/TASVideos/BizHawk/blob/master/README.md#gnulinux-and-macos)
+* [Building](https://github.com/TASVideos/BizHawk/blob/master/README.md#building)
+* [Usage](https://github.com/TASVideos/BizHawk/blob/master/README.md#usage)
+	* [TASing](https://github.com/TASVideos/BizHawk/blob/master/README.md#tasing)
+	* [Cores](https://github.com/TASVideos/BizHawk/blob/master/README.md#cores)
+* [Support and troubleshooting](https://github.com/TASVideos/BizHawk/blob/master/README.md#support-and-troubleshooting)
+* [Contributing](https://github.com/TASVideos/BizHawk/blob/master/README.md#contributing)
+	* [Testing](https://github.com/TASVideos/BizHawk/blob/master/README.md#testing)
+* [Related projects](https://github.com/TASVideos/BizHawk/blob/master/README.md#related-projects)
+* [License](https://github.com/TASVideos/BizHawk/blob/master/README.md#license)
 
-[Developer build](https://ci.appveyor.com/project/zeromus/bizhawk-udexo/build/artifacts) of the most recent commit
+## Features and systems
 
-Never mix different versions of BizHawk. Keep each version in its own folder.
+The BizHawk common features (across all cores) are:
+* format and region detection for game images
+* image corruption warning checked against database
+* 10 save slots with hotkeys and ∞ named savestates
+* speed control, including frame stepping and rewinding
+* memory view/search/edit in all parts of the emulated HW
+* input recording (making TAS movies)
+* screenshotting and recording video
+* organised firmware
+* input, framerate, and other overlays
+* emulated controllers via a comprehensive input mapper
```

Comment:
Add "hotkey bindings to control the UI" as another item.

Reply:
done